v1_0/assets/tasks.pbtxt

```diff
@@ -0,0 +1,205 @@
+task {
+  name: "Image classification"
+  dataset {
+    name: "Imagenet classification validation set"
+    type: IMAGENET
+    path: "/sdcard/mlperf_datasets/imagenet/img"
+    groundtruth_src: "/sdcard/mlperf_datasets/imagenet/imagenet_val_full.txt"
+  }
+  lite_dataset {
+    name: "Open images subset for classification"
+    type: IMAGENET
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/imagenet.zip"
+    groundtruth_src: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/assets/imagenet_val.txt"
```

Comment:
Here and below:
https://github.com/mlcommons/mobile_models/raw/main/v1_0/assets/ does not contain files except for tasks.pbtxt What the reason to use links to non-existent files?